README.md

```diff
@@ -1,5 +1,5 @@
 # Nextcloud Contacts
-![Downloads](https://img.shields.io/github/downloads/nextcloud/contacts/total.svg?style=flat-square)
+![Downloads](https://img.shields.io/github/downloads/nextcloud/contacts/total?style=flat-square)
```

Comment:
why? :thinking: :grin:

Reply:
No clue where that came from. I think my git has a personality.